gene2phenotype_project/gene2phenotype_app/serializers/user.py

```diff
@@ -187,21 +200,24 @@ def create(self, validated_data):
       
         panels = validated_data.pop('panels', []) # popping panels because i do not need it for creation
         user = User.objects.create_user(**validated_data)
```

Comment:
This is not related to this PR, but I just noticed the 'date_joined' is not being set when we create a user.
You could send the date here. 
We have a method to get the correct date format in utils -> get_date_now()